test/lib/processors/nonAsciiEscaper.js

```diff
@@ -0,0 +1,101 @@
+const test = require("ava");
+
+const nonAsciiEscaper = require("../../../lib/processors/nonAsciiEscaper");
+
+/**
+ * Executes string escaping. Returns <code>undefined</code> if nothing was escaped.
+ *
+ * @param {string} input string
+ * @param {Object} [options]
+ * @returns {Promise<string|undefined>} escaped string if non-ascii characters present, <code>undefined</code> otherwise
+ */
+const escape = async function(input, options) {
+	let result = undefined;
+	const resource = {
```

Comment:
Why don't you just create a real Resource? https://sap.github.io/ui5-tooling/api/module-@ui5_fs.Resource.html

It's not clear to me what became easier with this approach.

Reply:
done